Sdk/Speckle.Converters.Common.DependencyInjection/ContainerRegistration.cs

```diff
@@ -23,6 +23,7 @@ This will require consolidating across other connectors.
     builder.AddScoped<IFactory<IToHostTopLevelConverter>, Factory<IToHostTopLevelConverter>>();
     builder.AddScoped<IConverterResolver<IToHostTopLevelConverter>, ConverterResolver<IToHostTopLevelConverter>>();
 
+    builder.AddScoped<LocalToGlobalConverterUtils>();
```

Comment:
Could we revert back to injecting this just in connectors that need it, rather than in all?

Reply:
makes sense

Reply:
done